templates/module/Source/module/Private/Generated/Monitor/loggingdecorator.cpp.tpl

```diff
@@ -84,8 +84,15 @@ public:
 {{$Class}}::{{$Class}}()
 	: I{{$Iface}}Interface()
 {
+}
+
+{{$Class}}::~{{$Class}}() = default;
+
+void {{$Class}}::Initialize(FSubsystemCollectionBase& Collection)
```

Comment:
I would call Collection -> Subsystems

Reply:
I just followed the naming as in the engine code